bands/templates/band_admin/band_info.html

```diff
@@ -11,183 +11,184 @@
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
+<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {% csrf_token %}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel_name %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Name:</label>
-            <div class="col-sm-6">
-                {{ band.panel_name }}
-            </div>
+    {% if band.panel.name %}
+    <div class="row">
+        <label class="col-sm-2 control-label">Panel Name:</label>
+        <div class="col-sm-6">
+            {{ band.panel.name }}
         </div>
+    </div>
     {% endif %}
 
-    {% if band.panel_length %}
-        <div class="row">
-            <label class="col-sm-2 control-label">Panel Length:</label>
-            <div class="col-sm-6">
-                {{ band.panel_length }}
-            </div>
+    {% if band.panel.length %}
+    <div class="row">
+        <label class="col-sm-2 control-label">Panel Length:</label>
+        <div class="col-sm-6">
+            {{ band.panel.length }}
         </div>
+    </div>
```

Comment:
Oh wow, whoops. This is indeed not intended. Will keep that tactic in mind for the future.